log/roller.go

```diff
@@ -88,19 +90,34 @@ func (l Roller) GetLogWriter() io.Writer {
 	if err != nil {
 		absPath = l.Filename // oh well, hopefully they're consistent in how they specify the filename
 	}
+
+	// try get logger
+	lumberjacksLocker.RLock()
```

Comment:
```suggestion
	lumberjacksLocker.Lock()
	defer lumberjacksLocker.Unlock()
```
Maybe we can just add these two lines? it will make the code looks simpler.
I don't think it's necessary to introduce the `getSlowLogWriter` method.

Reply:
Here is a performance optimization for the scenario of more reads and less writes. Of course, can remove it if you think it is not necessary.

Reply:
Thanks, got it.

Reply:
The scenario that call the `GetLogWriter` method  is Infrequent, and we will save the `io.Writer` in the start phase, so I agree with doujiang24.